ru_RU.lang

```diff
@@ -209,35 +209,35 @@ replaymod.gui.rendersettings.addtoqueue=Добавить в очередь
 replaymod.gui.rendersettings.renderer=Метод Рендера
 
 replaymod.gui.rendersettings.renderer.default=Стандартный
-replaymod.gui.rendersettings.renderer.stereoscopic=Стереоскопическая
+replaymod.gui.rendersettings.renderer.stereoscopic=Стереоскопич.
 replaymod.gui.rendersettings.renderer.cubic=Кубический
-replaymod.gui.rendersettings.renderer.equirectangular=Равнопромежуточная
+replaymod.gui.rendersettings.renderer.equirectangular=Равноугольный
 replaymod.gui.rendersettings.renderer.ods=ODS
-replaymod.gui.rendersettings.renderer.blend=Экспорт в Blender [Экспериментально]
+replaymod.gui.rendersettings.renderer.blend=Экспорт в Blender [БЕТА]
```

Comment:
Does "Beta" have a different connotation in Russian? Note that I would not want to call this feature Beta in English, it is very much only experimental, it doesn't even work with modern Blender versions, nor with modern Minecraft versions.

Reply:
It's just shorter and obvious by context that it refers to replaymod function